packages/unified-latex-ctan/utils/argument-globber.ts

```diff
@@ -0,0 +1,72 @@
+import { arg } from "@unified-latex/unified-latex-builder";
+import { Argument, Node } from "@unified-latex/unified-latex-types";
+import { parse as parseArgspec } from "@unified-latex/unified-latex-util-argspec";
+import { gobbleSingleArgument } from "@unified-latex/unified-latex-util-arguments";
+import { match } from "@unified-latex/unified-latex-util-match";
+
+export function getOptionalArg(
```

Comment:
If these are going to be imported by other code, they should be re-exported in `index.ts`. They also need doc strings. Though, I think most of what you are doing can/should be accomplished in the existing `gobbleSingleArgument` function.